docs/source/author/content_style_guide.rst

```diff
@@ -3,3 +3,32 @@ Content Style Guide
 
 .. contents:: Contents
   :local:
+
+Writing Style
+==============================================================================
+
+When using pronouns in reference to a hypothetical person, gender neutral pronouns (they/their/them) should be used.
+
+All documents (other than those for internal use only) must be written clearly and simply so that a non-expert is able to understand them.
+Preferably documents should be readable by students.
+
+Any jargon used needs to be clearly explained and should be considered as a glossary definition.
+
+Capitalisation Rules
+------------------------------------------------------------------------------
+In the majority of cases capitalisation should not be used for keywords and titles.
+
+The following wouldn't be capitalised:
+binary number(s), digits, binary digits
+
+Extra notes for specific content
+==============================================================================
+
+Glossary
+------------------------------------------------------------------------------
+
+The following are added to the glossary and linked to where the words are used:
```

Comment:
Give an example of the syntax for glossary entries, and where each entry should be put